sdk/core/azure-core/src/http/winhttp/win_http_transport.cpp

```diff
@@ -196,8 +196,25 @@ std::string GetHeadersAsString(Azure::Core::Http::Request const& request)
   return requestHeaderString;
 }
 
+Azure::Core::Context::Key NoClientCertificateConfiguration;
```

Comment:
Add a doc comment with the info about this key-value pair.
This context has bool true/false as the value.

Reply:
This is impl details, but sure, I'll do it

Reply:
> This context has bool true/false as the value.

The key is not related to the value-type.  There should be no relation between them.  The value-type is only known by the entity who creates the option and provides the factory method to create the context